core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -306,6 +306,9 @@ public AttributeFormat getAttributeFormat() {
                 true /* stored */, false /* tokenized */, false /* multivalued */, XML_TYPE));
         descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_URI, true /* indexed */,
                 true /* stored */, false /* tokenized */, false /* multivalued */, STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_DOWNLOAD_URL,
+                true /* indexed */, true /* stored */, false /* tokenized */,
+                false /* multivalued */, STRING_TYPE));
```

Comment:
This confuses me.  If you want to include a dynamic action value (aka the download URL) as a metacard attribute, the value shouldn't indexed, or stored, in fact I don't think it should be included in the Metacard BasicTypes definition at all.  

Assuming it doesn't make sense to be included in the BasicTypes definition, then that backsup that point that the design for including the download URL as a Metacard attribute is flawed and that the download-url should not really be a Metacard attribute, but rather an action that can be derived from a Metacard (which it currently is today, so I am still confused by the desire to include it as an attribute as I think it leads to more confusion and more complex logic). 

Also wouldn't you then have to update the endpoint code to specifically use this value?  I didn't see that in this PR.